.configs/.sourcery.yaml

```
@@ -0,0 +1,83 @@
+# You can use it to get Sourcery working in the way you want, such as
+# ignoring specific refactorings, skipping directories in your project,
+# or writing custom rules.
+
+# 📚 For a complete reference to this file, see the documentation at
+# https://docs.sourcery.ai/Configuration/Project-Settings/
+
+# This file was auto-generated by Sourcery on 2024-07-19 at 16:11.
+
+version: '1'  # The schema version of this config file
+
+ignore: # A list of paths or files which Sourcery will ignore.
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)